CasterTrainer.py

```
@@ -0,0 +1,322 @@
+    for name, tb in TEXT_INPUTS.items():
+        try:
+            SPELL_CAPS[name] = int(tb.Text.strip()) if tb.Text else 0
+        except Exception:
+            SPELL_CAPS[name] = 0
+    _update_gump("")
+    if any(v > 0 for v in SPELL_CAPS.values()):
+        RUNNING = True
+
+
+# Render or refresh the gump UI.
+def _update_gump(current):
```

Ruff (0.14.14)

[warning] Unused function argument: `current`

(ARG001)